examples/react/jsx-layers/README.md

```diff
@@ -1,4 +0,0 @@
-# JSX-layers
```

Comment:
This change is part of the JSX layers PR, better land that one first and rebase...